projects/iio_demo/readme.txt

```diff
@@ -3,3 +3,7 @@ cat sample.dat | iio_writedev -u serial:/dev/ttyUSB0,921600 -b 400  demo_device
 Read from device:
 iio_readdev -u serial:/dev/ttyUSB0,921600 -b 400 -s 6400 demo_device > sample.dat
 
+Windows:
+Get-Content ascii.dat | iio_writedev -u serial:COM9,921600 -b 100 -s 100 demo_device_output
```

Comment:
nitpick: Since Get-Content is a PowerShell widget, maybe it would be a good idea to specify that it should be used in powershell.